package_policies/M1.md

```diff
@@ -1,14 +1,10 @@
-**M1.** Each xSDK-compatible package must support either the **community GNU Autoconf or CMake
-options**<sup>3</sup> as defined in the following document:
+**M1.** Each xSDK-compatible package must support portable installation through Spack.
 
-[xSDK Community Installation Policies: GNU Autoconf and CMake Options](./community-installation-policies.md).
+All configuration, build, and installation phases must be specified in a [Spack](https://spack.io/)
```

Comment:
Should say "All configuration, build, and installation phases must be specified in a [Spack](https://spack.io/) **package**".

Reply:
Wording is currently:
All configuration, build, and installation phases must be specified in a Spack package recipe.

Reply:
I see that now, missed it in the diff.